src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import todos from '../../api/todos';
+
+type Props = {
+  users: User[],
+  onSubmit: (todo: Todo) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({ users, onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [isTitleEmpty, setIsTitleEmpty] = useState(false);
+  const [isUserChosen, setIsUserChosen] = useState(true);
+  const maxId = Math.max(...todos.map(todo => todo.id));
```

Comment:
You calculate this value inside the component but use it only inside the function `handleAddOfTodo`

It makes sense to calculate it inside `handleAddOfTodo` - only when it is needed